content/en/core/releases/feature_releases.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Feature Releases"
+linkTitle: "Feature Releases"
+weight: 2
+no_list: true
+description: >
+    Feature Releases for the CHT Core Framework
+relatedContent: >
+    core/releases
+    contribute#are-you-a-partner-wondering-how-issues-are-prioritized
+    
+---
+
+The Product Team has two dedicated sub-teams (Allies and Care Teams) who try to deliver the most impactful features at a pace that is faster than our regular release cycle. To this end, select partners working directly with the Product Team can access what are called "Feature Releases". 
```

Comment:
Given the broader audience of the docs site this intro needs some more context, and here's a suggestion that may provide higher level intro without getting into internal processes.

```suggestion
To build and iterate on new features at a pace that is faster than our regular release cycle, some features are released in a _Feature Release_. Feature Releases are based on the most recent release and only include improvements related to a feature being developed. These releases are tested to be production-ready so that new features can be studied with CHT partners in a live deployment, with the aim of getting the feature ready for wider use in an upcoming release.
```